README.md

```diff
@@ -65,15 +73,128 @@ href="https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tr
 ## Instructions
```

Comment:
Why do we need these instructions?

Reply:
removed in 5ab064bf9e45ea280a7f72d103b9aec6e3fdfae2